docs/02-concepts/01-data-processing/06-k-member-micro-aggregations.md

```diff
@@ -0,0 +1,224 @@
+| 64  |XS   |black    |73                |
+| 12  |XL   |brown    |16                |
+| 79  |XS   |red      |186               |
+| 82  |M    |black    |259               |
+| 23  |L    |red      |134               |
+| 21  |L    |brown    |46                |
+
+## Micro-Aggregations Batch Job
+
+Now that the data contract is defined correctly (make sure the contract is
+[active](/docs/02-concepts/02-data-contracts/index.md#states)), define the configuration for the
+micro-aggregations batch job. We assume you have created a [data connector](/docs/02-concepts/04-data-connectors.md)
+in the same project. The configuration file should then look like this:
+
+```json showLineNumbers
+{
+  #callout-1
+  "source_data": { 
+    "data_connector_ref": { "name": "dataConnectorName"},
+    "path_prefix": "input",
+    "file_name": "file.csv",
+    "data_type": { "csv": { "charset": "UTF-8" } }
+  },
+  #callout-2
+  "target_data": {
+    "data_connector_ref": { "name": "dataConnectorName"},
+    "path_prefix": "output",
+    "file_name": "file.csv",
+    "data_type": { "csv": { "charset": "UTF-8" } }
+  },
+  #callout-3
+  "data_contract_ref": {
+    "handle": "handle",
+    "name": "name",
+    "version": "xx.xx.xx"
+  },
+  #callout-4
+  "aggregation_config": {
+    "minimum_k_anonymity": 3
+  }
+}
+```
+
+1. The source of the data by the means of your data connector, the file name and path prefix and the type of
+   the data (only `csv` is possible at this point)
+2. The target location of the data, similar to the source data
+3. The reference to the data contract applied to the data
+4. The minimum amount of _`k`_-anonymity you want to ensure
+
+Finally, you can run the micro-aggregations batch job by running the following command:
+
+```shell
+strm create batch-job --type micro-aggregation --file my_config.json
+```
+
+The results of our example are found below:
+
+<table>
+<tr><th>Before</th><th>After</th></tr>
+<tr><td>
+
+| age |size |hairColor|transactionAmount |
+|-----|-----|---------|------------------|
+| 33  |XS   |red      |123               |
+| 16  |S    |brown    |46                |
+| 64  |XS   |black    |73                |
+| 12  |XL   |brown    |16                |
+| 79  |XS   |red      |186               |
+| 82  |M    |black    |259               |
+| 23  |L    |red      |134               |
+| 21  |L    |brown    |46                |
+
+</td><td>
+
+|age               |size |hairColor|transactionAmount |
+|------------------|-----|---------|------------------|
+|32.33333333333333 |XS   |red      |141.33333333333334|
+|16.25             |S    |brown    |49.250000000000014|
+|66.0              |XS   |black    |88.4              |
+|8.000000000000007 |XL   |brown    |7.333333333333314 |
+|80.0              |XS   |red      |174.75            |
+|82.66666666666666 |M    |black    |269.0             |
+|28.25             |L    |red      |139.25            |
+|20.666666666666664|L    |brown    |53.333333333333314|
```

Comment:
This output data set may be confusing: there is no k-anonymity in this subset. Could we use a different subset so that the After table is k-anonymous?

Reply:
great call!